swagger.yml

```diff
@@ -2415,6 +2415,34 @@ components:
           x-go-name: Smoothing
       type: object
       x-go-package: github.com/TykTechnologies/tyk/user
+    SessionEndpoint:
+      title: SessionEndpoint holds rate limit on endpoint level.
+      type: object
+      properties:
+        path:
+          type: string
+        methods:
+          type: array
+          items:
+            $ref: '#/components/schemas/SessionEndpointMethod'
+    SessionEndpointMethod:
+      title: SessionEndpointMethod holds rate limit on endpoint method level.
+      properties:
+        name:
+          type: string
+        limit:
+          $ref: '#/components/schemas/SessionEndpointRateLimit'
+      type: object
+    SessionEndpointRateLimit:
+      title: SessionEndpointMethod holds rate limit on endpoint method level.
```

Comment:
```suggestion
      title: SessionEndpointRateLimit holds rate limit on endpoint method level.
```

Reply:
Just use https://github.com/TykTechnologies/exp/tree/main/scripts/model2schema to verify the rest. Should match godoc and should be able to generate the full schema for the definition from the model